wordpress-mobile/WordPress-Android
# Issue/9703 site creation background

### Fix
Update the background color from `neutral_50`/`gray_50` (`#e1e2e2`) to `neutral_0`/`gray_0` (`#f6f6f6`) for multiple site creation views as described in https://github.com/wordpress-mobile/WordPress-Android/issues/9703.  See the screenshots below for illustration.

![site_creation_background](https://user-images.githubusercontent.com/3827611/56683811-31935980-6683-11e9-9515-ceb997aa429f.png)

### Test
1. Go to ***Sites*** tab.
2. Tap ***Switch Site*** button in top card view.
3. Tap ***Add New Site*** button in top app bar.
4. Tap ***Create WordPress.com Site*** option in dialog.
5. Tap any segment on ***Create Site*** screen.
6. Notice ***Skip*** button container background.
7. Tap ***Skip*** button in ***1 of 3*** screen.
8. Notice ***Skip*** button container background.
9. Tap ***Skip*** button in ***2 of 3*** screen.
10. Enter "test" in ***Search Domains*** field.
11. Notice ***No available addresses matching your search*** background.
12. Enter domain available in ***Search Domains*** field.
13. Tap domain in list.
14. Notice ***Create Site*** button container background.
15. Tap ***Create Site*** button.
16. Notice ***We're creating your new site*** background.
17. Wait for site to be created.
18. Notice preview container background.
19. Notice site address bar background.
20. Notice ***OK*** button container background.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.
